tests/integrations/raftstore/test_stats.rs

```diff
@@ -534,58 +587,67 @@ pub fn test_rollback() {
     ));
 }
 
-fn test_query_num(query: Box<Query>, enable_ttl: bool) {
-    let (mut cluster, client, ctx) = must_new_and_configure_cluster_and_kv_client(|cluster| {
+fn test_query_num<Api: APIVersion>(query: Box<Query>, is_raw_kv: bool) {
+    let (mut cluster, client, mut ctx) = must_new_and_configure_cluster_and_kv_client(|cluster| {
         cluster.cfg.raft_store.pd_store_heartbeat_tick_interval = ReadableDuration::millis(50);
         cluster.cfg.split.qps_threshold = 0;
         cluster.cfg.split.split_balance_score = 2.0;
         cluster.cfg.split.split_contained_score = 2.0;
         cluster.cfg.split.detect_times = 1;
         cluster.cfg.split.sample_threshold = 0;
-        cluster.cfg.storage.enable_ttl = enable_ttl;
+        cluster.cfg.storage.set_api_version(Api::TAG);
     });
+    ctx.set_api_version(Api::CLIENT_TAG);
 
-    let k = b"key".to_vec();
+    let mut k = b"key".to_vec();
     // When a peer becomes leader, it can't read before committing to current term.
     // Force a successful read to wait for the correct timing.
     cluster.must_get(&k);
     let store_id = 1;
-    if enable_ttl {
-        raw_put(&cluster, &client, &ctx, store_id, k.clone());
+    if is_raw_kv {
+        k = b"r\0key".to_vec();
+        raw_put::<Api>(&cluster, &client, &ctx, store_id, k.clone());
     } else {
+        k = b"x\0key".to_vec();
         put(&cluster, &client, &ctx, store_id, k.clone());
     }
     let region_id = cluster.get_region_id(&k);
     query(ctx, &cluster, client, store_id, region_id, k.clone());
 }
 
-fn test_delete_query() {
-    let k = b"key".to_vec();
+fn test_raw_delete_query<Api: APIVersion>() {
+    let k = b"r\0key".to_vec();
```

Comment:
What is `r\0`?

Reply:
`r` is the key prefix of RawKV, and `\0` is the key space id in earlier design...
Let me remove the `\0` to avoid being confusing.

P.S. The `r` will refactor by some constant or macro to eliminate the hard-coding later.

Reply:
PTAL~